libopenage/gamestate/team.cpp

```diff
@@ -61,4 +64,12 @@ member_type Team::get_member_type(Player &player) {
 	return member_type::none;
 }
 
+std::vector<const Player*> Team::get_players() const {
```

Comment:
Ah, sorry. I think there's no better way to return dict keys separately, if the dict would support returning its keys, the internal implementation would be exactly this.

http://www.lonecpluspluscoder.com/2015/08/13/an-elegant-way-to-extract-keys-from-a-c-map/

If you like, you can create the mentioned template function to our `libopenage/util/` library.